experimental/dds/tree2/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -99,8 +99,7 @@ export interface ChangeEvents<TChangeset> {
 export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends SharedObject<
 	TransformEvents<ISharedTreeCoreEvents, ISharedObjectEvents>
 > {
-	private readonly editManager: EditManager<TChange, ChangeFamily<TEditor, TChange>>;
-	private readonly undoRedoManager: UndoRedoManager<TChange, TEditor>;
+	protected readonly editManager: EditManager<TChange, ChangeFamily<TEditor, TChange>>;
```

Comment:
Can we please keep this private, and directly expose the `localBranchUndoRedoManager` to `SharedTree` via a protected property, rather than handing it the whole `editManager`?